test/extended/prometheus/prometheus.go

```diff
@@ -319,17 +319,24 @@ func bringUpPrometheusFromTemplate(oc *exutil.CLI) (ns, host, bearerToken string
 	}
 
 	waitForServiceAccountInNamespace(oc.AdminKubeClient(), "kube-system", "prometheus", 2*time.Minute)
```

Comment:
If you're waiting for a service account, why wouldn't you also wait for its secret?  Seems like this helper could wait for both in testing.

Reply:
This shouldn't even be using its own method, there's a common helper.  Will clean up in 3.10